src/Analytics/Adapter/Plausible.php

```diff
@@ -140,7 +140,7 @@ public function validate(Event $event): bool
 
         $validateURL = $this->endpoint.'/v1/stats/aggregate?'.http_build_query([
             'site_id' => $this->domain,
-            'filters' => json_encode(['goal' => $event->getName()]),
+            'filters' => json_encode(['goal' => $event->getType()]),
```

Comment:
Just double checking, is this on purpose?

Reply:
Yeah, I noticed it was wrong with how plausible's API works